bool line_right
bool line_middle
bool line_left